### PR TITLE
Kuba styling

### PR DESCRIPTION
**Pull request by KubaTuta**
I did bunch of styling, tweaked some values, remove some which were doing nothing, template rows are now setting automatically so less @media.
Now after click on pagi, site should stay on the bottom, tell me if it is good or bad idea.

### CODE REVIEW

File: src/common/Navigation/styled.js [hunk 1]
```diff
@@ -1,48 +1,78 @@
...
 `;
 
 export const NavBoxLine = styled.div`
   display: flex;
   align-items: center;
-    @media(max-width: ${({ theme }) => theme.breakpoint.small}) {
-      flex-basis: 90%;
-    } 
+  @media(max-width:${({ theme }) => theme.breakpoint.medium}) {
+    padding-left: 40px;
```
Comment (Mateusszzp) [inline src/common/Navigation/styled.js:41]: it looks good, but if the responsiveness is set to medium 1052px, I would add the top margin a little glued to the top

File: src/Features/People/Core/Success/Tile/styled.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    font-weight: 500;
+    font-size: 14px;
+  };
+  @media(max-width: ${({ theme }) => theme.breakpoint.mobile}) {
+    max-width: 136px;
+    max-height: 245px;
+    font-weight: 500;
+    font-size: 14px;
+  };
+`;
```
Comment (Mateusszzp) [inline src/Features/People/Core/Success/Tile/styled.js:81]: now the tiles are holding up very well, you've worked hard

**Review by Mateusszzp — APPROVED** [review]
The tiles with the videos lay out well and the navigation is nice too.
I think you need to improve the top margin in responsiveness
This is what it looks like
good job

File: src/Features/Movies/Core/Success/Tile/styled.js
Comment (HaniaNassalska) [inline src/Features/Movies/Core/Success/Tile/styled.js:1]: I love what you did! It looks so much better! :) :)
I noticed that some of the posters are smaller than others. I thought about that. The Poster component could be moved into another div. Something like that:
```
export const PosterWrapper = styled.div`
  min-width: 292px;
  min-height: 434px;
  margin-bottom: 8px;
   
  @media(max-width: ${({ theme }) => theme.breakpoint.small}) {
    min-width: 114px;
    min-height: 169px;
  } 
`

export const Poster = styled.img`
  width: 100%;
  height: 434px;
  border-radius: 5px;
  object-fit: cover;
  object-position: top;
   
  @media(max-width: ${({ theme }) => theme.breakpoint.small}) {
    height: 169px;
    width: 114px;
  };
`;
```

 What do you think about that? Maybe you have a better idea?
I also noticed that you wrote the same value in small and mobile size. It is not necessary, I think.

**Review by HaniaNassalska — COMMENTED** [review]
I think the issue with poster sizes needs to be fixed before merging.

**Pull request merged**